lib/util.ts

```diff
@@ -1080,7 +1080,6 @@ export class PoolConvUtil {
     if (!autoPad) {
       return;
     }
-
     if (pads.length !== 2 * (inputDims.length - 2)) {
```

Comment:
>     [](start = 0, length = 4)

Nit: blank change. Consider revert this file.

Reply:
done.